components/Chips/src/MDCChipView.h

```diff
@@ -0,0 +1,169 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface MDCChipView : UIControl
+
+/*
+ A UIImageView that leads the title label.
+
+ Only visible when the image is non-nil.
+ */
+@property(nonatomic, readonly, nonnull) UIImageView *imageView;
+
+/*
+ A UIImageView that leads the title label. Appears above the imageView.
+
+ Only visible when the chip is selected and the image is non-nil.
+ */
+@property(nonatomic, readonly, nonnull) UIImageView *selectedImageView;
+
+/*
+ A UIView that trails the title label.
+
+ It will be given a size based on the value returned from sizeThatFits:.
+ */
+@property(nonatomic, strong, nullable) UIView *accessoryView;
+
+/*
+ The title label.
+
+ @note The title color is controlled by setTitleColor:forState:.
+ */
+@property(nonatomic, readonly, nonnull) UILabel *titleLabel;
+
+/*
+ Padding around the image view. Only used if the image view has a non-nil image.
+
+ Defaults to (0, 0, 0, 0).
+ */
+@property(nonatomic, assign) UIEdgeInsets imagePadding UI_APPEARANCE_SELECTOR;
```

Comment:
Are `imagePadding`, `titlePadding`, and `accessoryPadding` included or excluded in the value returned by `intrinsicContentSize`?

Reply:
They are included in `intrinsicContentSize` and `sizeThatFits`.

Reply:
Awesome. I think it's worth adding this to the document as is done for [UIButton.contentEdgeInsets](https://developer.apple.com/documentation/uikit/uibutton/1624036-contentedgeinsets).

> The chip uses this property to determine intrinsicContentSize and sizeThatFits(_:).

Reply:
Good idea. Done.